### PR TITLE
(DOCSP-26022): Add TSQ Set examples, move subqueries

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-26022

### Staged Changes

- [Filter Data - Swift SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-26022/sdk/swift/crud/filter-data/#set-operators)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/swift/crud/filter-data.txt [hunk 1]
```diff
@@ -344,20 +322,45 @@ to a single value.
...
+   .. literalinclude:: /examples/generated/code/start/QueryEngine.snippet.tsq-set-operators.swift
+      :language: swift
+
+.. _ios-realm-swift-query-api-subquery:
+
+Subqueries
+~~~~~~~~~~
+
+You can iterate through a collection property with another query using a
+subquery. To form a subquery, you must wrap the expression in parenthesis
```
Comment (cbullinger) [inline source/sdk/swift/crud/filter-data.txt:347]:
```suggestion
subquery. To form a subquery, you must wrap the expression in parentheses
```
should be plural?

File: source/sdk/swift/crud/filter-data.txt [hunk 2]
```diff
@@ -811,3 +777,40 @@ the object.
...
+``SUBQUERY()`` predicate function. ``SUBQUERY()`` has the following signature:
+
+.. code-block:: objective-c
+
+   SUBQUERY(<collection>, <variableName>, <predicate>)
+
+- ``collection``: the name of the list property to iterate through
+- ``variableName``: a variable name of the current element to use in the subquery
+- ``predicate``: a string that contains the subquery predicate. You can use the
+  variable name specified by ``variableName`` to refer to the currently-iterated
```
Comment (cbullinger) [inline source/sdk/swift/crud/filter-data.txt:796]:
```suggestion
  variable name specified by ``variableName`` to refer to the currently iterated
```

**Review by cbullinger — APPROVED** [review]
lgtm

**Comment by dacharyc** [comment]
Huh, it looks like the iOS CI is failing after the most recent commit, which just added some grammar fixes. I'm going to merge anyway because [the prior commit](https://github.com/mongodb/docs-realm/pull/2494/commits/e3fb436ec011f6a3544414daabc1f46e363c0222) that actually had the test changes passed in CI. I'll look into this failure more later.

**Pull request merged**